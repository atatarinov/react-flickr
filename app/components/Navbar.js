import React from 'react';

export default function Navbar(props) {

  const { handleChange, text, handleSubmit } = props;
  return (
    <div className="container-fluid">
    <h1 id="main-title">flickr</h1>
    <form className="form-group" onSubmit={handleSubmit}>
      <label className="form-container">
        <input
          type="text"
          className="form-control"
          name="name"
          placeholder="Find your photos"
          value={text}
          onChange={handleChange}
        />
      </label>
      <button type="button" className="btn btn-light" id="submit">Submit</button>
    </form>
    </div>
  );
}
