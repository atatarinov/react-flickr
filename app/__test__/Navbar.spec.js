import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

test('Navbar renders correnctly', () => {
  const component = renderer.create(<Navbar />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
