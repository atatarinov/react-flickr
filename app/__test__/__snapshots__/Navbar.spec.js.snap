// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correnctly 1`] = `
<div
  className="container-fluid"
>
  <h1
    id="main-title"
  >
    flickr
  </h1>
  <form
    className="form-group"
    onSubmit={undefined}
  >
    <label
      className="form-container"
    >
      <input
        className="form-control"
        name="name"
        onChange={undefined}
        placeholder="Find your photos"
        type="text"
        value={undefined}
      />
    </label>
    <button
      className="btn btn-light"
      id="submit"
      type="button"
    >
      Submit
    </button>
  </form>
</div>
`;
